components/style/config.rs

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /*
+ * servo_config is a crate in components/config that pulls in a lot of random stuff. In gecko mode,
```

Comment:
Rustdoc uses `//!` for module or crate level documentation. Could you convert `/* .. */` to that  please? With this change you can remove the `#[allow(missing_docs)]` attribute in the 
`#[allow(missing_docs)] pub mod config;` line. Also we will be able to see this comment in the doc.servo.org after this.

nit: also it would be wonderful if you can shorten these lines to match license length while you're here.